compiler/README.md

```diff
@@ -0,0 +1,70 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
+
+# Introduction
+
+This document discusses the ONE compiler. ONE compiler analyses, quantizes and optimizes the input model. It takes many model formats and converts to a common [Circle](#circle-format)  format which is then deployed.
+
+<!-- Contents -->
+# Contents 
+* [Build Setup](#Build-Setup)
+* [Supported Formats](#Supported-Nerual-Network-Formats)
+* [Model Analysis](#Model-Analysis)
+* [Data Representation & Quantization](#Data-Representation-And-Quantization)
+* [Circle Format](#Circle-Format)
+
+<!-- Build Setup -->
+## Build Setup
+<!-- System Requirments -->
+* System Setup   
+This setup is to build the ONE compiler on Ubuntu. Its tested on Ubuntu 16.04 as well as Ubuntu 18.04.  
+Get the code [Samsung ONE](https://github.com/Samsung/ONE) from Github. 
+
+<!-- Software/Package Requirements -->
+* Dependencies
+    * Cmake (Greater than or equal to 3.1)
+    * g++ toolchain ( Greater than or equal to 4.8)
```

Comment:
Hmm... AFAIK, g++ should be higher than 5.
I am still using ubuntu 16.04 but encountered some compile errors because the version of `g++` was too low.
So, after I changed to `g++` version 6, error didn't occurred.

Did you success with 4.8 or 5.x to compile?

Reply:
No, i tested with 8+. No worries, will fix this.